examples/layer-browser/src/utils/grid-aggregator.js

```diff
@@ -27,7 +27,7 @@ export function pointsToWorldGrid(points, worldUnitSize) {
     return accu;
   }, {});
 
-  const maxHeight = Math.max.apply(null, Object.values(gridHash));
+  const maxHeight = Math.max.apply(null, Object.keys(gridHash).map(k => gridHash[k]));
```

Comment:
Yea, I am using iterate in the src code. This is for examples. there are only around 500 cells.